File: Python/Easy/Filter/Sorting/2037__Minimum_Number_of_Moves_to_Seat_Everyone.py
def minMovesToSeat(seats: list[int], students: list[int]) -> int:
    seats.sort()
    students.sort()
    total = 0
    for i, j in zip(students, seats):
        total += abs(i - j)
    return total


# seats = [3, 1, 5]
# students = [2, 7, 4]

# seats = [4,1,5,9]
# students = [1,3,2,6]
seats = [2, 2, 6, 6]
students = [1, 3, 2, 6]
print(minMovesToSeat(seats, students))

File: PHP/Easy/Filters/Sorting/2037__Minimum_Number_of_Moves_to_Seat_Everyone.php
<?php
function minMovesToSeat($seats, $students)
{
    sort($seats);
    sort($students);

    $total = 0;
    for ($i = 0; $i < count($students); $i++) {
        $total += abs($students[$i] - $seats[$i]);
    }
    return $total;
}
$seats = [3,1,5];
$students = [2,7,4];

echo minMovesToSeat($seats, $students);